README.md

```diff
@@ -169,10 +198,10 @@ Neither of these are official, however many users have reported success with the
 | Library                                                           | Language   | Description/Notes                                                                     |
 | ----------------------------------------------------------------- | ---------- | ------------------------------------------------------------------------------------- |
 | [aiosu](https://github.com/niceaesth/aiosu)                       | python     | Async Python library for everything osu! related                                      |
-| [ossapi](https://github.com/circleguard/ossapi)                   | python     | Python library for interacting with the osu! API.                                     |
-| [osu-api-extended](https://github.com/cyperdark/osu-api-extended) | typescript |                                                                                       |
+| [ossapi](https://github.com/circleguard/ossapi)                   | python     | A definitive python wrapper for the osu! api.                                         |
+| [osu-api-extended](https://github.com/cyperdark/osu-api-extended) | typescript | Package for advanced work with "osu" api                                              |
+| [rosu](https://github.com/MaxOhn/rosu)                            | rust       | A rust wrapper for osu!apiv1.                                                         |
```

Comment:
I was thinking something like this:
```suggestion
| [rosu](https://github.com/MaxOhn/rosu) ([api v2](https://github.com/MaxOhn/rosu-v2))                              | rust       | A rust wrapper for osu!api v1/v2.                                                         |
```

Reply:
fixed in https://github.com/CirclePeople/osu-list/pull/9/commits/190e462c1c81909c4e5e99a0a1f153d037c4eb4b